include.xml

```diff
@@ -4,7 +4,7 @@
 	
 	<assets path="assets/sounds" include="*.ogg" if="html5" unless="unit-test" />
 	<assets path="assets/sounds" include="*.mp3" if="flash" embed="true" />
-	<assets path="assets/sounds" include="*.ogg" unless="flash || html5" embed="true" />
```

Comment:
Was this causing issues / does lime legacy not support embedding .oggs?

If so, we should at least embed them when `lime_legacy` is defined.